packages/datagrid/src/components/DefaultHeaderRenderer/MenuDropDownQualityBar.scss

```diff
@@ -0,0 +1,45 @@
+.button-menu-dropdown {
```

Comment:
dropdown menu has nothing to do in datagrid, you should expose onClick and refs as asked in the JIRA

Reply:
i understand, i have fix it